cibyl/sources/elasticsearch/api.py

```diff
@@ -164,6 +166,50 @@ def get_last_build(self: object, builds_jobs: AttributeDictValue):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_object)
 
+    def get_deployment(self, **kwargs):
+        """Get deployment information for jobs from elasticsearch server.
+
+        :returns: container of jobs with deployment information from
+        elasticsearch server
+        :rtype: :class:`AttributeDictValue`
+        """
+
+        if 'job_name' in kwargs:
+            jobs_to_search = kwargs.get('job_name').value
+        else:
+            jobs_to_search = kwargs.get('jobs').value
```

Comment:
What if 'jobs' is also not in kwargs?

Reply:
Thank you Jorge. I've changed this and now it works fine. It will get all the jobs with topology as Jenkins is doing.